src/ppx/grfx/grfx_swapchain.cpp

```diff
@@ -143,6 +129,67 @@ void Swapchain::Destroy()
     grfx::DeviceObject<grfx::SwapchainCreateInfo>::Destroy();
 }
 
+Result Swapchain::CreateColorImages()
+{
+    if (!mColorImages.empty()) {
+        return ppx::SUCCESS;
+    }
+    for (uint32_t i = 0; i < mCreateInfo.imageCount; ++i) {
+        grfx::ImageCreateInfo rtCreateInfo = ImageCreateInfo::RenderTarget2D(mCreateInfo.width, mCreateInfo.height, mCreateInfo.colorFormat);
+        rtCreateInfo.ownership             = grfx::OWNERSHIP_RESTRICTED;
+        rtCreateInfo.RTVClearValue         = {0.0f, 0.0f, 0.0f, 0.0f};
+        rtCreateInfo.initialState          = grfx::RESOURCE_STATE_PRESENT;
+        rtCreateInfo.usageFlags =
+            grfx::IMAGE_USAGE_COLOR_ATTACHMENT |
```

Comment:
nit: maybe this bit-list could use the same format as the other? (`imageCreateInfo.usageFlags.bits.transferSrc = true`, or the other way around?

Reply:
yeah, i found it is a bit weird as well. Also would it be better to have the default values defined in 1 place so that in the future if someone need to make a change, there is no need to modify in 2 functions?